packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -495,8 +494,11 @@ export class ProjectLifecycleManager {
       d.currentTestingType = testingType
       d.wizard.chosenBundler = null
       d.wizard.chosenFramework = null
-      if (d.currentProjectData) {
-        d.currentProjectData.testingTypeData = makeTestingTypeData(testingType)
+      if (testingType) {
```

Comment:
same thought as below, should we be resetting the diagnostics state here? maybe warnings for the current testing type

might be worth leaving to [setCurrentProject](https://github.com/cypress-io/cypress/pull/23034/files#diff-63757aa782fe9ed7c93e5f91cd7afdcf8f617d2b4eea12d1f51d09747fb28247R410) and [clearCurrentProject](https://github.com/cypress-io/cypress/pull/23034/files#diff-d36f6626bf805d1bbbfba4786ca3a518f0083613a9814fd037c5592808949606R84)?

Reply:
I tried deleting this but it breaks a few things. Test case that fails is

1. open mode
2. choose project
3. config errors
4. click the "projects" breadcrumb to go back to global mode

If we do not clear the error here, you cannot get back to the projects page (error state overlay prevents you from seeing your projects).